scarb-metadata/src/command/metadata_command.rs

```diff
@@ -61,7 +61,9 @@ impl MetadataCommand {
     /// Creates a default `scarb metadata` command, which will look for `scarb` in `$PATH` and
     /// for `Scarb.toml` in the current directory or its ancestors.
     pub fn new() -> Self {
-        Self::default()
+        let mut c = Self::default();
+        c.env_remove("SCARB_UI_VERBOSITY");
+        c
```

Comment:
Hmm, this looks like a hack. Wouldn't it be more correct if `scarb metadata` outputted metadata, no matter what verbosity is set?